src/corelibs/U2View/src/ov_msa/MaEditorWgt.cpp

```diff
@@ -85,13 +100,9 @@ QWidget* MaEditorWgt::createHeaderLabelWidget(const QString& text, Qt::Alignment
                              proxyMouseEventsToNameList);
 }
 
-MaEditorStatusBar* MaEditorWgt::getStatusBar() const {
-    return statusBar;
-}
-
-void MaEditorWgt::initWidgets() {
+void MaEditorWgt::initWidgets(bool addStatusBar, bool addOverviewArea) {
     setContextMenuPolicy(Qt::CustomContextMenu);
-    setMinimumSize(300, 200);
+    setMinimumSize(300, 100);
```

Comment:
Why do we need this change? There is a chance that there will be no space for sequence area at all (we need some space for consensus)

Reply:
200 is too big for multiline view in case of few seqs
this is a minimum size, not maximum